util/src/test/java/tk/zielony/randomdata/util/StringVersionGeneratorText.java

```diff
@@ -0,0 +1,25 @@
+package tk.zielony.randomdata.util;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by kirinpatel on 12/1/17.
+ */
+
+public class StringVersionGeneratorText {
+
+    @Test
+    public void generationIsCorrect() throws Exception {
+        StringVersionGenerator versionNumberGenerator = new StringVersionGenerator();
+        int version = versionNumberGenerator.nextVersionNumber(null);
```

Comment:
avoid passing nulls as parameters. If this parameter is not used, remove it. If it's not used in some cases, provide an overloaded method with no parameter.